addons/core/translations/resources/en-us.yaml

```diff
@@ -627,6 +629,9 @@ role:
       descendants:
         label: Add all descendants
         help: Add all of the orgs and the projects in the orgs that are underneath this scope.
+    remove-org:
+      title: Remove projects under this org?
+      description: Do you want to remove the projects under this org from this role? Or do you just want to remove the org from this role?
```

Comment:
I could be reading it wrong but when I read the first question, it feels to me like we can remove just the projects but keep the org selected, not remove the projects AND the org.